game/views.py

```
@@ -1135,16 +1135,20 @@ def leaderboard_view(request):
     stats, _ = PuzzleStats.objects.get_or_create(
         user=request.user
     )
 
-    stats.puzzles_solved = data.get("puzzles_solved", 0)
-    stats.current_streak = data.get("current_streak", 0)
-    stats.best_streak = data.get("best_streak", 0)
-    stats.daily_completions = data.get("daily_completions", 0)
+    stats.puzzles_solved += 1
+    stats.current_streak += 1
+    if stats.current_streak > stats.best_streak:
+        stats.best_streak = stats.current_streak
```

Ruff (0.15.15)

[warning] Replace `if` statement with `max` call

Replace with `max` call

(PLR1730)